### ISSUE TITLE
[BUG] Crash, "forestry-wood-fertiliser-00" (recipe): Difficulty normal: amount must be larger than 0

### ISSUE THREAD

**Issue by ZwerOxotnik** [Deadlock989/IndustrialRevolution#427]
```txt
   1.508 Loading mod settings IndustrialRevolution3 3.1.4 (settings.lua)
   1.509 Loading mod settings IndustrialRevolution3 3.1.4 (settings-updates.lua)
   1.513 Loading mod core 0.0.0 (data.lua)
   1.557 Loading mod base 1.1.107 (data.lua)
   1.758 Loading mod IndustrialRevolution3 3.1.4 (data.lua)
   1.893 Loading mod base 1.1.107 (data-updates.lua)
   1.908 Loading mod IndustrialRevolution3 3.1.4 (data-updates.lua)
   1.968 Loading mod IndustrialRevolution3 3.1.4 (data-final-fixes.lua)
   3.053 Checksum for core: 607699937
   3.054 Checksum of base: 1054619811
   3.054 Checksum of IndustrialRevolution3Assets1: 0
   3.054 Checksum of IndustrialRevolution3Assets2: 0
   3.054 Checksum of IndustrialRevolution3Assets3: 0
   3.054 Checksum of IndustrialRevolution3Assets4: 0
   3.054 Checksum of IndustrialRevolution3: 491945504
   3.270 Error ModManager.cpp:1625: Error while loading recipe prototype "forestry-wood-fertiliser-00" (recipe): Difficulty normal: amount must be larger than 0 in property tree at ROOT.recipe.forestry-wood-fertiliser-00.ingredients[1]
Modifications: Industrial Revolution 3
```

- Factorio 1.1.107
- Industrial Revolution 3.1.4

**Comment by Deadlock989** [comment]
You are 20 versions behind the latest release. A breaking API change in Factorio 1.1.101 required a fix in version 3.1.20 of IR3. Any previous version is not compatible with recent versions of Factorio.

**Comment by Deadlock989** [comment]
Duplicate of #358
